sdk/storage/azure-storage-blob/samples/blob_samples_client_side_encryption_keyvault.py

```diff
@@ -96,7 +96,7 @@ def get_kid(self):
 client_secret = get_env_var(CLIENT_SECRET)  # aka password in AzureCLI
```

Comment:
Should axe this (and the lines above containing `tenant_id` and `client_id`), we shouldn't be using any of these in any capacity anymore

Reply:
Thanks for putting that out! I slightly overlooked this file.

Please see the fix regarding this :D